### DISCUSSION TITLE
How would I go about running a modded .jar file?

### DISCUSSION THREAD

**Discussion by nwill111** [AlexProgrammerDE/balena-minecraft-server#156] [Q&A] [answered]
Im trying to host a fabric server I was previously running on my PC. I moved all the server files over to the serverfiles folder but mc-server insists on downloading and running a vanilla jar. 

Thanks for the help!

**Comment by nwill111** [comment, marked as answer]
Disregard. Figured it out! It is by setting the JAR_FILE variable to the name of the modded jar file

**Comment by viedok** [comment]
Hey! I guess you can now mark your question as resolved :)
